server/opts.go

```diff
@@ -255,6 +265,14 @@ func (o *Options) ProcessConfigFile(configFile string) error {
 				}
 				o.Users = auth.users
 			}
+			// Check for nkeys
+			if auth.nkeys != nil {
+				if o.Users != nil {
+					return fmt.Errorf("Can not have users when nkeys are also defined.")
```

Comment:
I thought we would support both in case we have users that use client libs that don't support nkeys.

Reply:
I was thinking we either do nkey or accounts or users/tokens but not a mix. Account mode is nkeys but with JWT while nkey mode doesn't use JWT from the client in my version.

Reply:
JWT is next revision where move from permissions being defined in server to being passed into server from client. We do support mixed mode when the server defines both. At least I thought I added a test for the. Can double check.

Reply:
I do have a test but it sets up server struct directly, so I will add a test for mixed mode from a config file, which I bet will not work, so will fix.